pyproject.toml

```diff
@@ -58,6 +58,7 @@ services = [
 hdf5 = [
     "h5py",
     "hdf5plugin",
+    "msgpack",
```

Comment:
msgpack is a dependency of python-blosc2.  Is this really necessary to include it here?

Reply:
I prefer to make it explicit since the affected Caterva2 code uses msgpack directly (not just through python-blosc2).